extensions/wikia/CuratedContent/maintenance/SpecialPromoteToCuratedContent.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * Script to upload image and description from Special:Promote to Curated Content
+ * Usage: run in context of a given wiki
+ * SERVER_ID=wiki_id php SpecialPromoteToCuratedContentMigrator.php
+ *
+ * @ingroup Maintenance
+ */
+
+require_once( dirname( __FILE__ ) . '/../../../../maintenance/Maintenance.php' );
+
+class SpecialPromoteToCuratedContentMigrator extends Maintenance {
+	public function __construct() {
+		$this->addOption( 'dry-run', 'Dry run - do not save changes to curated content' );
+		parent::__construct();
+	}
+
+	public function execute() {
+		global $wgCityId, $wgLang, $wgDBname;
+
+		$dryRun = $this->hasOption('dry-run');
+
+		$change = false;
+		$cv = new CityVisualization();
+		$communityDataService = new CommunityDataService( $wgCityId );
+
+		$curatedContentData = [
+			'curated' => $communityDataService->getCurated(),
+			'featured' => $communityDataService->getFeatured(),
+			'optional' => $communityDataService->getOptional(),
+			'community_data' => $communityDataService->getCommunityData()
+		];
+
+		if ( empty( $curatedContentData['community_data']['description'] ) ) {
+			$originalDescription = $curatedContentData['community_data']['description'];
```

Comment:
shouldn't be this assignment in else and the one from else here? Otherwise I don't understand something :p